### PR TITLE
Onboarding recoveryservicesbackup for autogeneration

### PR DESCRIPTION
**Pull request by hunmix**
```
> azure-schema-generator@1.0.0 generate-single D:\code\azure-resource-manager-schemas\generator
> ts-node cmd/generatesingle "recoveryservicesbackup/resource-manager"

[C:\Users\ADMINI~1\AppData\Local\Temp\schm_azspc] executing: git fsck
[C:\Users\ADMINI~1\AppData\Local\Temp\schm_azspc] executing: git reset -q .
[C:\Users\ADMINI~1\AppData\Local\Temp\schm_azspc] executing: git checkout -q -- .
[C:\Users\ADMINI~1\AppData\Local\Temp\schm_azspc] executing: git clean -q -fd
[C:\Users\ADMINI~1\AppData\Local\Temp\schm_azspc] executing: git gc
[C:\Users\ADMINI~1\AppData\Local\Temp\schm_azspc] executing: git fetch -q
[C:\Users\ADMINI~1\AppData\Local\Temp\schm_azspc] executing: git checkout -q origin/master
Using whitelist config:
{
  "basePath": "recoveryservicesbackup/resource-manager",
  "namespace": "Microsoft.RecoveryServices",
  "suffix": "Backup"
}
[D:\code\azure-resource-manager-schemas\generator] executing: autorest-beta.cmd --version=3.0.6257 --use=@autorest/azureresourceschema@3.0.79 --azureresourceschema --output-folder=C:\Users\ADMINI~1\AppData\Local\Temp\qbu63e5j0f9 --tag=all-api-versions --api-version=2016-06-01 --title=none --pass-thru:subset-reducer C:\Users\ADMINI~1\AppData\Local\Temp\schm_azspc\specification\recoveryservicesbackup\resource-manager\readme.md
AutoRest code generation utility [cli version: 3.0.6187; node: v12.16.1, max-memory: 8192 gb]
(C) 2018 Microsoft Corporation.
https://aka.ms/autorest
   Loading AutoRest core      'C:\Users\Administrator\.autorest\@autorest_core@3.0.6257\node_modules\@autorest\core\dist' (3.0.6257)
   Loading AutoRest extension '@autorest/azureresourceschema' (3.0.79->3.0.79)
[21.98 s] Generation Complete
================================================================================================================================
Filename: D:\code\azure-resource-manager-schemas\schemas\2016-06-01\Microsoft.RecoveryServices.Backup.json
Provider Namespace: Microsoft.RecoveryServices
API Version: 2016-06-01
Resource Types (Resource Group Scope):
- vaults/backupFabrics/protectionContainers/protectedItems
- vaults/backupPolicies
================================================================================================================================
[D:\code\azure-resource-manager-schemas\generator] executing: autorest-beta.cmd --version=3.0.6257 --use=@autorest/azureresourceschema@3.0.79 --azureresourceschema --output-folder=C:\Users\ADMINI~1\AppData\Local\Temp\q71f8f7ty4s --tag=all-api-versions --api-version=2016-08-10 --title=none --pass-thru:subset-reducer C:\Users\ADMINI~1\AppData\Local\Temp\schm_azspc\specification\recoveryservicesbackup\resource-manager\readme.md
AutoRest code generation utility [cli version: 3.0.6187; node: v12.16.1, max-memory: 8192 gb]
(C) 2018 Microsoft Corporation.
https://aka.ms/autorest
   Loading AutoRest core      'C:\Users\Administrator\.autorest\@autorest_core@3.0.6257\node_modules\@autorest\core\dist' (3.0.6257)
   Loading AutoRest extension '@autorest/azureresourceschema' (3.0.79->3.0.79)
[15.52 s] Generation Complete
[D:\code\azure-resource-manager-schemas\generator] executing: autorest-beta.cmd --version=3.0.6257 --use=@autorest/azureresourceschema@3.0.79 --azureresourceschema --output-folder=C:\Users\ADMINI~1\AppData\Local\Temp\0ew6knih7j7d --tag=all-api-versions --api-version=2016-12-01 --title=none --pass-thru:subset-reducer C:\Users\ADMINI~1\AppData\Local\Temp\schm_azspc\specification\recoveryservicesbackup\resource-manager\readme.md
AutoRest code generation utility [cli version: 3.0.6187; node: v12.16.1, max-memory: 8192 gb]
(C) 2018 Microsoft Corporation.
https://aka.ms/autorest
   Loading AutoRest core      'C:\Users\Administrator\.autorest\@autorest_core@3.0.6257\node_modules\@autorest\core\dist' (3.0.6257)
   Loading AutoRest extension '@autorest/azureresourceschema' (3.0.79->3.0.79)
[19.05 s] Generation Complete
================================================================================================================================
Filename: D:\code\azure-resource-manager-schemas\schemas\2016-12-01\Microsoft.RecoveryServices.Backup.json
Provider Namespace: Microsoft.RecoveryServices
API Version: 2016-12-01
Resource Types (Resource Group Scope):
- vaults/backupFabrics/protectionContainers
- vaults/backupstorageconfig
================================================================================================================================
[D:\code\azure-resource-manager-schemas\generator] executing: autorest-beta.cmd --version=3.0.6257 --use=@autorest/azureresourceschema@3.0.79 --azureresourceschema --output-folder=C:\Users\ADMINI~1\AppData\Local\Temp\w749se7bpc --tag=all-api-versions --api-version=2017-07-01 --title=none --pass-thru:subset-reducer C:\Users\ADMINI~1\AppData\Local\Temp\schm_azspc\specification\recoveryservicesbackup\resource-manager\readme.md
AutoRest code generation utility [cli version: 3.0.6187; node: v12.16.1, max-memory: 8192 gb]
(C) 2018 Microsoft Corporation.
https://aka.ms/autorest
   Loading AutoRest core      'C:\Users\Administrator\.autorest\@autorest_core@3.0.6257\node_modules\@autorest\core\dist' (3.0.6257)
   Loading AutoRest extension '@autorest/azureresourceschema' (3.0.79->3.0.79)
[16.86 s] Generation Complete
================================================================================================================================
Filename: D:\code\azure-resource-manager-schemas\schemas\2017-07-01\Microsoft.RecoveryServices.Backup.json
Provider Namespace: Microsoft.RecoveryServices
API Version: 2017-07-01
Resource Types (Resource Group Scope):
- vaults/backupFabrics/backupProtectionIntent
================================================================================================================================
[D:\code\azure-resource-manager-schemas\generator] executing: autorest-beta.cmd --version=3.0.6257 --use=@autorest/azureresourceschema@3.0.79 --azureresourceschema --output-folder=C:\Users\ADMINI~1\AppData\Local\Temp\lu1do7fombg --tag=all-api-versions --api-version=2018-12-20 --title=none --pass-thru:subset-reducer C:\Users\ADMINI~1\AppData\Local\Temp\schm_azspc\specification\recoveryservicesbackup\resource-manager\readme.md
AutoRest code generation utility [cli version: 3.0.6187; node: v12.16.1, max-memory: 8192 gb]
(C) 2018 Microsoft Corporation.
https://aka.ms/autorest
   Loading AutoRest core      'C:\Users\Administrator\.autorest\@autorest_core@3.0.6257\node_modules\@autorest\core\dist' (3.0.6257)
   Loading AutoRest extension '@autorest/azureresourceschema' (3.0.79->3.0.79)
[19.73 s] Generation Complete
[D:\code\azure-resource-manager-schemas\generator] executing: autorest-beta.cmd --version=3.0.6257 --use=@autorest/azureresourceschema@3.0.79 --azureresourceschema --output-folder=C:\Users\ADMINI~1\AppData\Local\Temp\wsou5ynic9f --tag=all-api-versions --api-version=2019-05-13 --title=none --pass-thru:subset-reducer C:\Users\ADMINI~1\AppData\Local\Temp\schm_azspc\specification\recoveryservicesbackup\resource-manager\readme.md
AutoRest code generation utility [cli version: 3.0.6187; node: v12.16.1, max-memory: 8192 gb]
(C) 2018 Microsoft Corporation.
https://aka.ms/autorest
   Loading AutoRest core      'C:\Users\Administrator\.autorest\@autorest_core@3.0.6257\node_modules\@autorest\core\dist' (3.0.6257)
   Loading AutoRest extension '@autorest/azureresourceschema' (3.0.79->3.0.79)
[22.08 s] Generation Complete
================================================================================================================================
Filename: D:\code\azure-resource-manager-schemas\schemas\2019-05-13\Microsoft.RecoveryServices.Backup.json
Provider Namespace: Microsoft.RecoveryServices
API Version: 2019-05-13
- vaults/backupFabrics/protectionContainers/protectedItems
- vaults/backupPolicies
================================================================================================================================
[D:\code\azure-resource-manager-schemas\generator] executing: autorest-beta.cmd --version=3.0.6257 --use=@autorest/azureresourceschema@3.0.79 --azureresourceschema --output-folder=C:\Users\ADMINI~1\AppData\Local\Temp\p29gevgpq7m --tag=all-api-versions --api-version=2019-06-15 --title=none --pass-thru:subset-reducer C:\Users\ADMINI~1\AppData\Local\Temp\schm_azspc\specification\recoveryservicesbackup\resource-manager\readme.md
AutoRest code generation utility [cli version: 3.0.6187; node: v12.16.1, max-memory: 8192 gb]
(C) 2018 Microsoft Corporation.
https://aka.ms/autorest
   Loading AutoRest core      'C:\Users\Administrator\.autorest\@autorest_core@3.0.6257\node_modules\@autorest\core\dist' (3.0.6257)
   Loading AutoRest extension '@autorest/azureresourceschema' (3.0.79->3.0.79)
[22.28 s] Generation Complete
================================================================================================================================
Filename: D:\code\azure-resource-manager-schemas\schemas\2019-06-15\Microsoft.RecoveryServices.Backup.json
Provider Namespace: Microsoft.RecoveryServices
API Version: 2019-06-15
Resource Types (Resource Group Scope):
- vaults/backupconfig
- vaults/backupFabrics/protectionContainers/protectedItems
- vaults/backupPolicies
================================================================================================================================
[D:\code\azure-resource-manager-schemas\generator] executing: autorest-beta.cmd --version=3.0.6257 --use=@autorest/azureresourceschema@3.0.79 --azureresourceschema --output-folder=C:\Users\ADMINI~1\AppData\Local\Temp\ezpurd5fr95 --tag=all-api-versions --api-version=2020-02-02 --title=none --pass-thru:subset-reducer C:\Users\ADMINI~1\AppData\Local\Temp\schm_azspc\specification\recoveryservicesbackup\resource-manager\readme.md
AutoRest code generation utility [cli version: 3.0.6187; node: v12.16.1, max-memory: 8192 gb]
(C) 2018 Microsoft Corporation.
https://aka.ms/autorest
   Loading AutoRest core      'C:\Users\Administrator\.autorest\@autorest_core@3.0.6257\node_modules\@autorest\core\dist' (3.0.6257)
   Loading AutoRest extension '@autorest/azureresourceschema' (3.0.79->3.0.79)
[15.58 s] Generation Complete
================================================================================================================================
Filename: D:\code\azure-resource-manager-schemas\schemas\2020-02-02\Microsoft.RecoveryServices.Backup.json
Provider Namespace: Microsoft.RecoveryServices
API Version: 2020-02-02
Resource Types (Resource Group Scope):
- vaults/privateEndpointConnections
================================================================================================================================
```

### CODE REVIEW

**Comment by hunmix** [comment]
regenerate
```
> azure-schema-generator@1.0.0 generate-single D:\code\azure-resource-manager-schemas\generator
> ts-node cmd/generatesingle "recoveryservicesbackup/resource-manager"

[C:\Users\hunmix\AppData\Local\Temp\schm_azspc] executing: git fsck
[C:\Users\hunmix\AppData\Local\Temp\schm_azspc] executing: git reset -q .
[C:\Users\hunmix\AppData\Local\Temp\schm_azspc] executing: git checkout -q -- .
[C:\Users\hunmix\AppData\Local\Temp\schm_azspc] executing: git clean -q -fd
[C:\Users\hunmix\AppData\Local\Temp\schm_azspc] executing: git gc
[C:\Users\hunmix\AppData\Local\Temp\schm_azspc] executing: git fetch -q
[C:\Users\hunmix\AppData\Local\Temp\schm_azspc] executing: git checkout -q origin/master
Using whitelist config:
{
  "basePath": "recoveryservicesbackup/resource-manager",
  "namespace": "Microsoft.RecoveryServices",
  "suffix": "resourceDefinitions/vaults"
}
[D:\code\azure-resource-manager-schemas\generator] executing: autorest-beta.cmd --version=3.0.6242 --use=@autorest/azureresourceschema@3.0.77 --azureresourceschema --output-folder=C:\Users\hunmix\AppData\Local\Temp\d6q2vx446o --tag=all-api-versions --api-version=2016-06-01 --title=none --pass-thru:subset-reducer C:\Users\hunmix\AppData\Local\Temp\schm_azspc\specification\recoveryservicesbackup\resource-manager\readme.md
AutoRest code generation utility [cli version: 3.0.6187; node: v10.15.3, max-memory: 8192 gb]
(C) 2018 Microsoft Corporation.
https://aka.ms/autorest
   Loading AutoRest core      'C:\Users\hunmix\.autorest\@autorest_core@3.0.6242\node_modules\@autorest\core\dist' (3.0.6242)
   Loading AutoRest extension '@autorest/azureresourceschema' (3.0.77->3.0.77)
[16.72 s] Generation Complete
================================================================================================================================
Filename: D:\code\azure-resource-manager-schemas\schemas\2016-06-01\Microsoft.RecoveryServices.resourceDefinitions\vaults.json
Provider Namespace: Microsoft.RecoveryServices
API Version: 2016-06-01
Resource Types (Resource Group Scope):
- vaults/backupFabrics/protectionContainers/protectedItems
- vaults/backupPolicies
================================================================================================================================
[D:\code\azure-resource-manager-schemas\generator] executing: autorest-beta.cmd --version=3.0.6242 --use=@autorest/azureresourceschema@3.0.77 --azureresourceschema --output-folder=C:\Users\hunmix\AppData\Local\Temp\o0n2xtukyxo --tag=all-api-versions --api-version=2016-08-10 --title=none --pass-thru:subset-reducer C:\Users\hunmix\AppData\Local\Temp\schm_azspc\specification\recoveryservicesbackup\resource-manager\readme.md
AutoRest code generation utility [cli version: 3.0.6187; node: v10.15.3, max-memory: 8192 gb]
(C) 2018 Microsoft Corporation.
https://aka.ms/autorest
   Loading AutoRest core      'C:\Users\hunmix\.autorest\@autorest_core@3.0.6242\node_modules\@autorest\core\dist' (3.0.6242)
   Loading AutoRest extension '@autorest/azureresourceschema' (3.0.77->3.0.77)
[8.37 s] Generation Complete
[D:\code\azure-resource-manager-schemas\generator] executing: autorest-beta.cmd --version=3.0.6242 --use=@autorest/azureresourceschema@3.0.77 --azureresourceschema --output-folder=C:\Users\hunmix\AppData\Local\Temp\2q1svkids3g --tag=all-api-versions --api-version=2016-12-01 --title=none --pass-thru:subset-reducer C:\Users\hunmix\AppData\Local\Temp\schm_azspc\specification\recoveryservicesbackup\resource-manager\readme.md
AutoRest code generation utility [cli version: 3.0.6187; node: v10.15.3, max-memory: 8192 gb]
(C) 2018 Microsoft Corporation.
https://aka.ms/autorest
   Loading AutoRest core      'C:\Users\hunmix\.autorest\@autorest_core@3.0.6242\node_modules\@autorest\core\dist' (3.0.6242)
   Loading AutoRest extension '@autorest/azureresourceschema' (3.0.77->3.0.77)
[11.84 s] Generation Complete
================================================================================================================================
Filename: D:\code\azure-resource-manager-schemas\schemas\2016-12-01\Microsoft.RecoveryServices.resourceDefinitions\vaults.json
Provider Namespace: Microsoft.RecoveryServices
API Version: 2016-12-01
Resource Types (Resource Group Scope):
- vaults/backupFabrics/protectionContainers
- vaults/backupstorageconfig
================================================================================================================================
[D:\code\azure-resource-manager-schemas\generator] executing: autorest-beta.cmd --version=3.0.6242 --use=@autorest/azureresourceschema@3.0.77 --azureresourceschema --output-folder=C:\Users\hunmix\AppData\Local\Temp\1yo11bradx6 --tag=all-api-versions --api-version=2017-07-01 --title=none --pass-thru:subset-reducer C:\Users\hunmix\AppData\Local\Temp\schm_azspc\specification\recoveryservicesbackup\resource-manager\readme.md
AutoRest code generation utility [cli version: 3.0.6187; node: v10.15.3, max-memory: 8192 gb]
(C) 2018 Microsoft Corporation.
https://aka.ms/autorest
   Loading AutoRest core      'C:\Users\hunmix\.autorest\@autorest_core@3.0.6242\node_modules\@autorest\core\dist' (3.0.6242)
   Loading AutoRest extension '@autorest/azureresourceschema' (3.0.77->3.0.77)
[9.69 s] Generation Complete
================================================================================================================================
Filename: D:\code\azure-resource-manager-schemas\schemas\2017-07-01\Microsoft.RecoveryServices.resourceDefinitions\vaults.json
Provider Namespace: Microsoft.RecoveryServices
API Version: 2017-07-01
Resource Types (Resource Group Scope):
- vaults/backupFabrics/backupProtectionIntent
================================================================================================================================
[D:\code\azure-resource-manager-schemas\generator] executing: autorest-beta.cmd --version=3.0.6242 --use=@autorest/azureresourceschema@3.0.77 --azureresourceschema --output-folder=C:\Users\hunmix\AppData\Local\Temp\fj7iw96h7uu --tag=all-api-versions --api-version=2018-12-20 --title=none --pass-thru:subset-reducer C:\Users\hunmix\AppData\Local\Temp\schm_azspc\specification\recoveryservicesbackup\resource-manager\readme.md
AutoRest code generation utility [cli version: 3.0.6187; node: v10.15.3, max-memory: 8192 gb]
(C) 2018 Microsoft Corporation.
https://aka.ms/autorest
   Loading AutoRest core      'C:\Users\hunmix\.autorest\@autorest_core@3.0.6242\node_modules\@autorest\core\dist' (3.0.6242)
   Loading AutoRest extension '@autorest/azureresourceschema' (3.0.77->3.0.77)
[12.61 s] Generation Complete
[D:\code\azure-resource-manager-schemas\generator] executing: autorest-beta.cmd --version=3.0.6242 --use=@autorest/azureresourceschema@3.0.77 --azureresourceschema --output-folder=C:\Users\hunmix\AppData\Local\Temp\tz7bsij44t8 --tag=all-api-versions --api-version=2019-05-13 --title=none --pass-thru:subset-reducer C:\Users\hunmix\AppData\Local\Temp\schm_azspc\specification\recoveryservicesbackup\resource-manager\readme.md
AutoRest code generation utility [cli version: 3.0.6187; node: v10.15.3, max-memory: 8192 gb]
(C) 2018 Microsoft Corporation.
https://aka.ms/autorest
   Loading AutoRest core      'C:\Users\hunmix\.autorest\@autorest_core@3.0.6242\node_modules\@autorest\core\dist' (3.0.6242)
   Loading AutoRest extension '@autorest/azureresourceschema' (3.0.77->3.0.77)
[15.41 s] Generation Complete
================================================================================================================================
Filename: D:\code\azure-resource-manager-schemas\schemas\2019-05-13\Microsoft.RecoveryServices.resourceDefinitions\vaults.json
Provider Namespace: Microsoft.RecoveryServices
API Version: 2019-05-13
Resource Types (Resource Group Scope):
- vaults/backupFabrics/protectionContainers/protectedItems
- vaults/backupPolicies
================================================================================================================================
[D:\code\azure-resource-manager-schemas\generator] executing: autorest-beta.cmd --version=3.0.6242 --use=@autorest/azureresourceschema@3.0.77 --azureresourceschema --output-folder=C:\Users\hunmix\AppData\Local\Temp\ajg91w6kklv --tag=all-api-versions --api-version=2019-06-15 --title=none --pass-thru:subset-reducer C:\Users\hunmix\AppData\Local\Temp\schm_azspc\specification\recoveryservicesbackup\resource-manager\readme.md
AutoRest code generation utility [cli version: 3.0.6187; node: v10.15.3, max-memory: 8192 gb]
(C) 2018 Microsoft Corporation.
https://aka.ms/autorest
   Loading AutoRest core      'C:\Users\hunmix\.autorest\@autorest_core@3.0.6242\node_modules\@autorest\core\dist' (3.0.6242)
   Loading AutoRest extension '@autorest/azureresourceschema' (3.0.77->3.0.77)
[15.72 s] Generation Complete
================================================================================================================================
Filename: D:\code\azure-resource-manager-schemas\schemas\2019-06-15\Microsoft.RecoveryServices.resourceDefinitions\vaults.json
Provider Namespace: Microsoft.RecoveryServices
API Version: 2019-06-15
Resource Types (Resource Group Scope):
- vaults/backupconfig
- vaults/backupFabrics/protectionContainers/protectedItems
- vaults/backupPolicies
================================================================================================================================
```

**Comment by anthony-c-martin** [comment]
I've added this suffix to conform with other suffixes, and '/' isn't an allowed character. Please could you re-run autogeneration and commit the changes?

File: schemas/2016-12-01/Microsoft.RecoveryServices.Backup.json [hunk 1]
```diff
@@ -0,0 +1,1031 @@
+{
```
Comment (anthony-c-martin) [inline schemas/2016-12-01/Microsoft.RecoveryServices.Backup.json:1]: Please could you remove `schemas/2016-12-01/Microsoft.RecoveryServices.json` as this file duplicates those resource types.

**Comment by hunmix** [comment]
merged with https://github.com/Azure/azure-resource-manager-schemas/pull/977

**Pull request closed without merging**